hack/delete.sh

```diff
@@ -14,6 +14,8 @@ fi
 USE_PROD_FLAG="-use-prod=false"
 if [[ -n "$TEST_IN_PRODUCTION" ]]; then
     USE_PROD_FLAG="-use-prod=true"
+else
```

Comment:
if you're adding `go generate` in create.sh, you should add it here as well

Reply:
Actually, this should point to `hack/fakerp.sh` I think

Reply:
done